knimin/templates/pm_plate_list.html

```diff
@@ -0,0 +1,173 @@
+<!-- The Plate Mapper module -->
+<!-- Plate list -->
+
+{% extends logged_in_index.html %}
+{% block head %}
+{% from json import dumps %}
+<script type="text/javascript" src="/static/js/labmin.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.validate.min.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<link rel="stylesheet" type="text/css" href="/static/vendor/css/jquery.dataTables.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
```

Comment:
I haven't done it because, maybe some of you prefer other icon sets? There are lots of files in an icon set. I am afraid that once merged it's hard to separate.